src/cc/frontends/clang/tp_frontend_action.cc

```diff
@@ -30,6 +30,7 @@
 
 #include "frontend_action_common.h"
 #include "tp_frontend_action.h"
+#include "../../../common.h"
```

Comment:
including "common.h" should be enough, I think.

Reply:
yep, I changed it